scripts/build/Dockerfile_solver_build

```diff
@@ -28,6 +24,22 @@ ARG Z3_VERSION
 
 WORKDIR $BASE
 
+#This can be a fixed llvm version, because solvers can use their own stdlib, needs to be
+#compiled with the same compiler probably, so clang-4.0
+COPY --from=llvmcxx ${BASE}/llvm-38-build_O_ND_A_libcxx/ ${BASE}/llvm-38-build_O_ND_A_libcxx/
```

Comment:
No fixed versions; solvers should be instrumented as well if needed, i.e. to debug issues how we invoke the solver from KLEE.

Reply:
yes? This enables that.